release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
+The Bug Triage role relates to both the Enhancements and CI Signal roles. Understanding the in-bound enhancements is important during the Early Release phase as they set the themes for incoming issues and bugs. Having an awareness on current test status is also critical, even though there is a specific lead for that area. The [documentation for CI Signal lead role](../ci-signal) includes a listing of special high risk test categories to monitor with useful information for the Bug Triage to also understand. The Bug Triage Lead should regularly interact with the peer leads for Enhancements and CI Signal.
 
-Bug Triage relates to both the Enhancements Lead and CI Signal Lead roles.  Understanding the in-bound enhancements is important during the Early Release phase as they set the themes for incoming issues and bugs.  Having an awareness on current test status is also critical, even though there is a specific lead for that area.  The [documentation for CI Signal lead role](../ci-signal) includes a listing of special high risk test categories to monitor with useful information for the Bug Triage Lead to also understand.  The Bug Triage Lead should regularly interact with the peer leads for Enhancements and CI Signal.
+Before starting, the Bug Triage members should refer to the following guides to get familiar with used labels:
+- [the documentation for issue `kind` labels](https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label)
+- [the documentation for defining priority and `priority` labels](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority).
 
 ### How to Escalate
-Whenever you find a bug that needs to be "fixed" or kicked out of the release, try the following escalation path:
 
-1. Leave a comment on the GitHub issue or PR: "This issue hasn't been updated in 3 months.  Are you still expecting to complete it for 1.17?".  It's helpful here to @ mention individuals or SIG ```-bugs``` or ```-pr-reviews``` aliases, e.g. "@kubernetes/sig-node-bugs" or "@kubernetes/sig-network-pr-reviews".
-2. Send a message to relevant SIG slack channels or mailing list about the problem: It's helpful to directly @ mention the relevant SIG Leads / Owners, and to condense multiple issues into a list, e.g. "Hey, these three issues haven't seen any attention, are they still valid for 1.17?"
-3. Message individual owners and reviewers directly via Slack and/or email (GitHub notification emails must have been filtered or missed if you're past step #1).  Individual's email addresses may be harder to find than GitHub ID, but are usually in the git commit history.  Sometimes Slack handles are hard to find.  There is no master list mapping human names to GitHub ID, email or Slack ID.  If you can't find contact info, asking in the appropriate SIG's Slack channel will usually get you pointed in the right direction.
```

Comment:
I have _never_ sent anyone an email while in this role. I'm +1 to remove the email parts.

Reply:
Many sig-leads do not spend much time on upstream Slack or Github, but instead have filters on their emails to occassionally check. There's a reason why mailing lists exist.

Reply:
I don't think I've ever had to resort to a direct email to an individual.  But I have used the "kubernetes-sig-${foo}-leads@googlegroups.com" mailing lists.  These lists are readily discoverable and it's a more reasonable path to go to the SIG chairs instead of trying to find the email address of an individual member of the SIG, especially when the only likely usage of these escalation paths is for time critical, exceptional situations just ahead of release.

Reply:
@tpepper Do we have some document explaining how these lists work that I can link here?

Reply:
The closest I think is this:
https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md#communicate

Reply:
@tpepper I've added a note about SIG leads mailing lists, PTAL.